CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/AnalyticsManager.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AnalyticsManager.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import Mixpanel
+
+class AnalyticsManager {
+    private static let mixPanelToken = "6262hhdfhdb929321222" // this id is fake we need to replace it
+    static var mixpanelInstance: MixpanelInstance?
+
+    static func initializeAnalytics() {
+        mixpanelInstance = Mixpanel.initialize(token: mixPanelToken, trackAutomaticEvents: false)
+        
+        // Identify the user with the deviceID
+        let deviceID = UIDevice.current.identifierForVendor?.uuidString
```

Comment:
[Clarification]: If I'm not mistaken, we will need to add additional info in our privacy manifest that we've gathered device data. 
Do we need to ask for tracking permissions in that case?

Reply:
No we don't need to ask for anything. 
"Mixpanel does not use IDFA, so it does not require user permission through the AppTrackingTransparency(ATT) framework."